api-samples/downloads/download_manager/service-worker.js

```diff
@@ -0,0 +1,219 @@
+chrome.downloads.setShelfEnabled(false);
+
+const colors = {
+  progressColor: '#0d0',
+  arrow: '#555',
+  danger: 'red',
+  complete: 'green',
+  paused: 'grey',
+  background: 'white'
+};
+
+Math.TAU = 2 * Math.PI; // http://tauday.com/tau-manifesto
```

Comment:
Hah, I didn't realise we had this.

Reply:
Tau? It's a recent thing. Someone observed that in all of geometry and physics they only place 2pi doesn't appear is in the formula for the area of a circle.